.changeset/fluffy-tips-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sap-ux/ui-components": patch
+"@sap-ux/prompting-ui": patch
```

Comment:
axios-extension should also be released as a patch.

Reply:
looks like we are adding changelogs to packages in `examples` - https://github.com/SAP/open-ux-tools/blob/main/examples/odata-cli/CHANGELOG.md
Then my previous comment is not valid

Reply:
Thanks @devinea Yes correct as the linting auto-correct bot pushed a commit. I thought the linting bot only works on the `main` branch.

Reply:
@devinea I guess patch bump of axios is not required now as there is only linting change.

Reply:
@hitesh-parmar I would release a change whenever there is a change to the `src` folder or dependencies.

Reply:
Done. Resolving this conversation.